code/modules/ruins/lavaland_ruin_code.dm

```diff
@@ -245,3 +245,77 @@
 		if(4)
 			new_spawn << "You wished for immortality, even as your friends lay dying behind you. No matter how many times you cast yourself into the lava, you awaken in this room again within a few days. You are overwhelmed with guilt."
 
+//Free Golems
+
+/obj/item/golem_shell
+	name = "empty golem shell"
+	icon = 'icons/obj/wizard.dmi'
+	icon_state = "construct"
+	desc = "The incomplete body of a golem."
+
+/obj/item/weapon/disk/design_disk/golem_shell
+	name = "Golem Creation Disk"
+	desc = "A gift from the Liberator."
+	icon_state = "datadisk1"
+	blueprint = /datum/design/golem_shell
+
+/datum/design/golem_shell
+	name = "Golem Shell Construction"
+	desc = "Allows for the construction of a Golem Shell."
+	id = "golem"
+	req_tech = list("materials" = 12)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 20000)
+	build_path = /obj/item/golem_shell
+	category = list("Misc")
```

Comment:
Nah it wasn't the category, your first reply helped me figure out what was wrong though, thanks